### DISCUSSION TITLE
HTML <title> for documentation pages

### DISCUSSION THREAD

**Discussion by roryashfordbentley** [GitbookIO/community#114] [0. General bugs and questions] [answered]
Good afternoon team! 

One of my colleagues pointed out earlier that the HTML page <title> element for docs are all 'Gitbook'. This makes sharing links more difficult on platforms like Slack because the actual title of the doc is missing. 

![Screenshot 2023-02-24 at 14 51 58](https://user-images.githubusercontent.com/878484/221209016-6de83acc-f158-4f2d-a93b-94fd1b174c33.png)

This also wouldn't pass [WCAG 2.4.2](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-navigation-mechanisms-title) - _Web pages have titles that describe topic or purpose. (Level A)_

**Comment by taranvohra** [comment, marked as answer]
Hey @roryashfordbentley, this is expected because this URL is protected. We have a `Slack` Integration in GitBook that solves this usecase by giving an accurate preview of the link that is shared in Slack
